gammapy/spectrum/tests/test_observation.py

```diff
@@ -190,7 +244,23 @@ def test_verify_npred(self):
 
         assert_allclose(npred_stacked.data.data, npred_summed)
 
-
+        
+    def test_stack_backscal(self):
```

Comment:
This test reimplements the functionality of ``SpectrumObservationStacker``, so it basically is a test if the two implementations are the same, not if they are correct.

A better way to set up tests is to come up with a test data set, for which we know what the test output should be and the assert to a given value, for example if you wanted to test
```
def add (a,b):
    return a + b
```
You would not setup the test like
```
val1 = 5
val2 = 10
result = val1 + val2
assert result == add(val1, va;2)
```
This test would pass even if the ``+`` operator in Python did something completely crazy, a better test would be
```
assert add(5, 10) == 15
```